app/services/cloud_topology_service.rb

```diff
@@ -0,0 +1,74 @@
+class CloudTopologyService < TopologyService
+  include UiServiceMixin
+
+  @provider_class = ManageIQ::Providers::CloudManager
+
+  def entity_type(entity)
+    entity.class.name.demodulize
+  end
+
+  def build_topology
+    topo_items = {}
+    links = []
+
+    included_relations = [
```

Comment:
Tried to extend relations as @Ladas suggest. It didn't work, so I made it like this:
```
included_relations = [
    :tags,
    :availability_zones => [:tags, :vms => :tags],
    :cloud_tenants => [:tags, :vms => :tags],
]
```